contracts/modules/AutoCompoundModule.sol

```diff
@@ -58,6 +60,7 @@ contract AutoCompoundModule is BaseModule {
         );
 
         (, bytes32 data) = IPositionManager(positionManagerAddress).getModuleInfo(tokenId, address(this));
+        require(data != bytes32(0), 'AutoCompoundModule::_checkIfCompoundIsNeeded: module data cannot be empty');
```

Comment:
Is this require really necessary? Didn't we already make sure that module data are not empty?

Reply:
where exactly @RiccardoGalbusera ?

Reply:
DoubleCheck, cause the data is passing in storage and we are not sure what we get